qemu/tests/migration.py

```diff
@@ -210,6 +211,14 @@ def check_dma():
 
             capabilities = ast.literal_eval(params.get("migrate_capabilities", "{}"))
             inner_funcs = ast.literal_eval(params.get("migrate_inner_funcs", "[]"))
+            mig_parameters = ast.literal_eval(params.get("migrate_parameters", "{}"))
+            target_mig_parameters = params.get("target_migrate_parameters", "{}")
+            target_mig_parameters = ast.literal_eval(target_mig_parameters)
```

Comment:
can be a single statement?, soft wrapped?

Reply:
It exceeded 80 char and the readability was not good, so I had split it in 2 lines

Reply:
okies..